test/unit/test-purifier.js

```diff
@@ -443,6 +453,33 @@ function runSanitizerTests() {
     });
   });
 
+  describe('should sanitize based on AMP doc format type', () => {
+    afterEach(() => {
+      html.removeAttribute('amp4Email');
+    });
+
+    it('should allow for input type file and password', () => {
+      // Given that the doc is not provided.
+      allowConsoleError(() => {
+        expect(purify('<input type="file">')).to.equal('<input type="file">');
+        expect(purify('<input type="password">'))
+            .to.equal('<input type="password">');
+      });
+    });
+
+    it('should disallow certain attributes on form for AMP4Email', () => {
```

Comment:
These attributes are not just form-related.

Reply:
reworded.